app/models/audio.rb

```diff
@@ -1,24 +1,7 @@
-# Generated via
-#  `rails generate hyrax:work Audio`
-class Audio < ActiveFedora::Base
+class Audio < Tufts::Curation::Audio
```

Comment:
I'm wondering if we can pull the entire model out.  This pattern still leaves us with having to ensure that each of the other applications has matching model code.  

I'm also worried about inadvertently getting unexpected behavior by having epigaea depend on a different version of Hyrax than, for instance, the Tufts Digital Library discovery application.